src/assets/js/main.js

```diff
@@ -124,6 +124,19 @@ function adjustToc() {
   $(document.body).scrollspy({ offset: 100, target: '#site-toc--side' });
 }
 
+function adjustHeadingAnchors() {
+  document.addEventListener('DOMContentLoaded', function() {
+    const headingAnchors = document.querySelectorAll('a.anchor[aria-hidden]');
```

Comment:
Thanks for thinking about this Tony! However, I'd prefer this specific change not land if possible as these anchors won't exist once https://github.com/dart-lang/site-www/pull/5483 lands, which should be sometime in the next few days.

If you can share more details about what the adjustment is intended to do, I can make sure it works with the new setup!

Reply:
@parlough : Can do. The issue is that the a11y check fails when it encounters the anchor links that appear next to each heading. This change removes those heading links from the tabindex, which allows the test to pass.